xml/Microsoft.Extensions.Logging.Console/ConsoleFormatter.xml

```diff
@@ -61,7 +61,7 @@
       </Docs>
     </Member>
     <Member MemberName="Write&lt;TState&gt;">
-      <MemberSignature Language="C#" Value="public abstract void Write&lt;TState&gt; (Microsoft.Extensions.Logging.Abstractions.LogEntry&lt;TState&gt;? logEntry, Microsoft.Extensions.Logging.IExternalScopeProvider scopeProvider, System.IO.TextWriter textWriter);" />
+      <MemberSignature Language="C#" Value="public abstract void Write&lt;TState&gt; (Microsoft.Extensions.Logging.Abstractions.LogEntry&lt;TState&gt; logEntry, Microsoft.Extensions.Logging.IExternalScopeProvider scopeProvider, System.IO.TextWriter textWriter);" />
```

Comment:
Check if this is expected.

Reply:
Looks good: https://github.com/dotnet/runtime/blob/6cf1b8ec012d52880d46fa4773f60ed52ddc9f3d/src/libraries/Microsoft.Extensions.Logging.Console/src/ConsoleFormatter.cs#L35